src/app/containers/Include/index.test.jsx

```diff
@@ -77,4 +85,44 @@ describe('IncludeContainer', () => {
     );
     expect(container).toMatchSnapshot();
   });
+
+  const runningIncludeTest = includeType => {
+    it(`should add require to the page for ${includeType}`, async () => {
+      render(
+        <IncludeContainerWithMockContext
+          toggleState={toggleStateFalse}
+          html={fakeMarkup}
+          type={includeType}
+        />,
+      );
+
+      await wait(() =>
```

Comment:
Should this be `waitFor`?
https://testing-library.com/docs/dom-testing-library/api-async#wait-deprecated-use-waitfor-instead